lib/hw_unique_key/Kconfig

```diff
@@ -4,13 +4,29 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-config HW_UNIQUE_KEY_LOAD
+config HW_UNIQUE_KEY
 	bool "Load the Hardware Unique Key in the secure bootloader"
-	depends on SOC_NRF52840
-	depends on IS_SECURE_BOOTLOADER
-	select FPROTECT_ENABLE_NO_ACCESS
+	depends on HAS_HW_NRF_CC310 || HAS_HW_NRF_CC312
```

Comment:
Is the prompt correct now that the functionality seems a bit more generic?

Reply:
No, fixed.